### DISCUSSION TITLE
Elmalar, Mahkumlar ve Matematik | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#126] [Site]
# Elmalar, Mahkumlar ve Matematik | SonsuzUs Github

Üç mantıkçı yanyana hücrelerde idam vaktini bekliyorlar. Bu sırada kral bunlara bir şans daha vermek için baş danışmanını gönderiyor ve danışman her bir hücreye en az bir en fazla da dokuz tane olacak şekilde farklı sayıda elma bırakıyor. Mantıkçılar sadece kendi hücrelerindeki elmaları görebiliyor, diğer hücrelerde yine aynı kurala göre elmalar bırakıldığını biliyorlar ama sayılarını bilmiyorlar. Sırayla her biri danışmana cevabı evet ya da hayır olan bir soru sorabilecek. Danışman da bu soruyu doğru cevaplayacak. Herkes soruları ve cevapları duyabilecek. Eğer bu sorular sonunda mantıkçılardan biri üç hücredeki toplam elma sayısını söyleyebilirse hepsi serbest kalacak.

https://sonsuzus.github.io/posts/elmalar/

**Comment by dgc6ybcbgq-ops** [comment]
20den büyük mü küçük mü?
